Ruby-Assignment-2-LoopingsAndLiterals/Question-1/solution2.rb

```diff
@@ -0,0 +1,31 @@
+
+TOTAL_CASES = gets.chomp.to_i
```

Comment:
Actually I was aware of it, but I thought of making it constant since it won't change during the course of the program.